web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/traversing/var/siblings.js

```diff
@@ -0,0 +1,17 @@
+define( function() {
+
+"use strict";
```

Comment:
:no_entry_sign: **Error**: Multi-line function call not indented correctly; expected 4 spaces but found 0 (*PEAR.Functions.FunctionCallSignature.Indent*).